udp-ballerina/connectionless_client.bal

```diff
@@ -16,14 +16,16 @@
 
 import ballerina/java;
 
-# Initializes the UDP connectionless client based on the provided configurations.
+# Initializes the UDP connectionless client based on the provided configurations
 #
-# + localHost - The local IP address string in textual presentation to which the socket is bound
+# + localHost - The local IP address string in textual presentation to which
+#                the socket is bound
 public client class Client {
  
-    private string? localHost = ();
+    // private string? localHost = ();
```

Comment:
If not required shall we remove this line?

Reply:
removed